.github/workflows/validate-doc-paths.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Check Metadata Linked Files'
+
+on: [pull_request]
+
+jobs:
+  verify-json-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: check for linked files
+        working-directory: './data'
+        run: node validate_metadata_docu_files.js
```

Comment:
Maybe, do that as a second step in the validate Workflow? It does make sense to only run this check if the schema is valid...

Reply:
totally makes sense, I updated it.